File: Rosalind/AC/HAMM.py
f = open('./input/rosalind_hamm.txt', 'r')
s=f.readline()
t=f.readline()

#res=0
#for i in range(len(s)):
#    if(s[i]!=t[i]):
#        res+=1
#print(res)

print(sum([a != b for a, b in zip(s, t)]))

File: Rosalind/AC/C++/HAMM.cpp
#include<iostream>
#include<string>

using namespace std;

int main(){
  string s1,s2;
  cin >> s1 >> s2;
  int res=0;
  for(int i=0;i<int(s1.size());++i)
    if(s1[i]!=s2[i])
      res++;
  cout << res << endl;
  
}
